api/v01/entities/vrp_result.rb

```diff
@@ -129,8 +129,20 @@ class VrpResultJob < Grape::Entity
       expose :graph, using: VrpResultJobGraphItem, documentation: { is_array: true, desc: 'Items to plot cost evolution' }
     end
 
+    class VrpResultVisualClusters < Grape::Entity
+      expose :vehicle, expose_nil: false, documentation: { desc: 'Vehicle partition visual' }
+      expose :work_day, expose_nil: false, documentation: { desc: 'Work_day partition visual' }
```

Comment:
my assumption is that a result should have a collection of partitions, which each have a name, a type and a feature collection

Reply:
In fact this 'collection' is represented by the fact that geojson is an array. Therefore we have different partitions according to each repetition. 

Each partition contains _vehicle_ and _work_day_ keys to represent each type of partition.

Reply:
In fact partitions were initially documented as being an array, which was a mistake. I think after discussing in one of our daily @braktar was okay with this.

geojsons are array to allow us to return one geojsons structure per repetition or per method.

(this comment appeared on wrong thread apparently)